docs/development.md

```diff
@@ -0,0 +1,21 @@
+# Development
+Additional information and help regarding developing `mgmt`. Useful tools, conventions, etc.
+
+Be sure to read [quick start guide](docs/quick-start-guide.md) first.
+
+## Testing
```

Comment:
This is cool to have in the docs, but please add some sort of line early on that says something along the lines of:

"Native golang tests are preferred over tests written in our shell testing framework. Please see https://golang.org/pkg/testing/ for more information."

Reply:
Will do

Reply:
Adding 'test-suites' is actually a bad pattern, shell tests should be preferred over testsuites and go tests should be preferred over them. I don't know what I was thinking when I wrote this. I'll adjust it to focus on gotest and shell test.